backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -72,13 +72,14 @@ def reshape_dataset_for_curation_api(dataset: dict) -> dict:
         del dataset["artifacts"]
     if dataset.get("processing_status"):
         dataset["processing_status"] = dataset["processing_status"]["processing_status"]
-    for ontology_element in DATASET_ONTOLOGY_ELEMENTS:
-        if dataset_ontology_element := dataset.get(ontology_element):
-            if not isinstance(dataset_ontology_element, list):
-                # Package in array
-                dataset[ontology_element] = [dataset_ontology_element]
-        else:
-            dataset[ontology_element] = []
+    if not preview:
+        for ontology_element in DATASET_ONTOLOGY_ELEMENTS:
+            if dataset_ontology_element := dataset.get(ontology_element):
+                if not isinstance(dataset_ontology_element, list):
+                    # Package in array
+                    dataset[ontology_element] = [dataset_ontology_element]
+            else:
+                dataset[ontology_element] = []
```

Comment:
This isn't going to work. There are [four elements returned in the dataset preview](https://github.com/chanzuckerberg/single-cell-data-portal/blob/5fa765b0a030a04ba28f9901e1e946bc85234bf5/backend/config/curation-api.yml#L678-L685) which need to have the reshaping logic applied to them.

Best to add a `DATASET_PREVIEW_ONTOLOGY_ELEMENTS` constant list and use it accordingly